Languages/IronPython/IronPythonCompiler/Program.cs

```diff
@@ -129,6 +129,34 @@ class Program {
             // variables for saving original working directory and return code of script
             var strVar = gen.DeclareLocal(u.Import(typeof(string)));
             var intVar = gen.DeclareLocal(u.Import(typeof(int)));
+            LocalBuilder dictVar = null;
+
+            if (config.PythonOptions.Count > 0) {
+                FieldInfo True = u.Import(typeof(ScriptingRuntimeHelpers)).GetField("True");
+                FieldInfo False = u.Import(typeof(ScriptingRuntimeHelpers)).GetField("False");
```

Comment:
I will change to ```var``` I think that is a good idea.

I used True and False since those are the names of the fields themselves. This is retrieving the field info for generating an access to that static field on ScriptingRuntimeHelpers in the emitted code.